src/js/controller/captions.js

```diff
@@ -37,10 +37,10 @@ const Captions = function(_model) {
                     _addTrack(track);
                     loadFile(track, (vttCues) => {
                         _addVTTCuesToTrack(track, vttCues);
-                    }, (error) => {
+                    }, (key, url, xhr, error) => {
                         this.trigger(ERROR, {
                             message: 'Captions failed to load',
-                            reason: error
+                            reason: error.code === 404 ? 'File not found' : key
```

Comment:
This event is supposed to be changed to a "warning". We shouldn't even be supporting "reason". The `error` object should be much more useful and is what was intended, but instead the key was being passed in because of a miss-understanding of the callback arguments. Make reason the error or remove it.